drift.py

```diff
@@ -1,140 +1,77 @@
 from helicase import Helicase
-from datetime import *
-import argparse
-import json, requests
+from datetime import datetime
+from spack import *
+import json, requests, argparse
 import subprocess, sys
 
 # Define SUCCESS for comparing command return codes.
 SUCCESS = 0
-args = None
 
 class Result:
```

Comment:
I was thinking about this after our meeting, and I think what you might want here is a dataclass: https://docs.python.org/3/library/dataclasses.html which will look more similar to what you originally had, and you can set defaults with the param definitions so you shouldn't need init.